content/en/agent/guide/agent-configuration-files.md

```diff
@@ -75,6 +75,8 @@ In order to provide a more flexible way to define the configuration for a check,
 
 `/datadog-agent/conf.d/<CHECK_NAME>.d/`.
 
+To prevent duplicate logs from being sent to Datadog, the agent does not accept multiple YAML files that point to the same log source. In the case where there is more than one YAML file that points to the same log source, the agent considers the files in alphabetical order and uses the first file.
```

Comment:
```suggestion
Note: For log collection, to prevent duplicate logs from being sent to Datadog, the agent does not accept multiple YAML files that point to the same log source. In the case where there is more than one YAML file that points to the same log source, the agent considers the files in alphabetical order and uses the first file.
```

Reply:
I think this note should go below, just above the array, otherwise it seems weird imho.